pkgs/epnix/support/gtest/default.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  epnixLib,
+  mkEpicsPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  fetchzip,
+  epnix,
+  local_config_site ? {},
+  local_release ? {},
+}:
+mkEpicsPackage {
+  pname = "gtest";
+  version = "1.0.1";
+  varname = "GTEST";
+
+  src = fetchFromGitHub {
+    owner = "epics-modules";
+    repo = "gtest";
+    rev = "c35f932";
+    hash = "sha256-Td7v74f+sOppbbEgEB944kKo2dWFipZ6bcM6CPpdUX0=";
```

Let's use the latest released version instead:

```suggestion
    rev = "v1.0.1";
    hash = "<...>";
```